app/src/androidTest/java/com/feedhenry/securenativeandroidtemplate/di/SecureApplicationTestModule.java

```diff
@@ -86,6 +87,11 @@ MobileCore provideMobileCore(Context context) {
         return mobileCore;
     }
 
+    @Provides @Singleton
```

Comment:
Hi @aidenkeating. Is there any chance you could add an asciidoc tag to this function? I just want to add some docs for where the security service is being initialized etc. eg.

```
// tag::securityServiceInit[]
@Provides @Singleton
SecurityService securityService(Context context, MobileCore mobileCore) {
   return mobileCore.getInstance(SecurityService.class);
}
// end::securityServiceInit[]
```

Reply:
@TommyJ1994 Just to confirm add tags to provideSecurityService in SecurityApplicationTestModule or SecurityApplicationModule?

Reply:
Just the `SecurityApplicationModule` is fine.